app/assets/stylesheets/custom_spacing_helpers.scss

```diff
@@ -16,4 +16,4 @@ $positions: ('top', 'left', 'bottom', 'right');
       padding-#{$position}: ($size + .5) + rem;
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Looks like one or more of us have a newline-on-last line configuration discrepancy.
I have mine setup to always include a newline, as advised [here](https://thoughtbot.com/blog/no-newline-at-end-of-file) and other places.

Reply:
i don't have a setting about it, but have started to add them by habit bc my coworker's linter is always adding them. happy to do whatever anyone wants!